src/styles/bookcard.module.css

```diff
@@ -0,0 +1,48 @@
+.bookCardContainer {
+  background-color: blueviolet;
+  max-width: 20rem;
+  border: 2px solid black;
+  border-radius: 10px;
+  padding-top: 10px;
+  padding: 5px;
+  position: relative;
+}
+
```

Comment:
styles should be defined along side their respective component declarations.
src/components/BookCard/BookCard.tsx
                                             /BookCard.module.css